### PR TITLE
Adjust content alerts for clean theme

### PR DESCRIPTION
**Pull request by nlisgo**

### CODE REVIEW

**Review by discodavey — APPROVED** [review]

**Comment by discodavey** [comment]
@nlisgo Can you have a look at the code and let me know if all is good?

**Comment by nlisgo** [comment]
> @nlisgo Can you have a look at the code and let me know if all is good?

I'm going to pull this into an instance of journal and we can see how well this covers the various stages of the content alerts interface.

**Comment by discodavey** [comment]
@nlisgo Can you review this change please. Chris H wanted the form to be centered at all times.

File: assets/sass/patterns/organisms/content-header-simple.scss [hunk 1]
```diff
@@ -19,3 +19,72 @@
...
+
+.clean {
+  @include margin(12, "bottom");
+
+  p {
+    color: $color-text;
+    font-family: $font-secondary;
+
+    a:not(.additional-asset__link--download):not(.asset-viewer-inline__download_all_link):not(.asset-viewer-inline__open_link):not(.reference__title):not(.doi):not(.reference__authors_link):not(.trigger):not(.popup__link):not(.see-more-link) {
+      border-bottom-style: solid;
```
Comment (discodavey) [inline assets/sass/patterns/organisms/content-header-simple.scss:31]: I thought that any main content links would be dotted. Underline looks very harsh. I think we need to talk to Chris about this?

**Pull request merged**